wifiphisher/common/interfaces.py

```diff
@@ -243,6 +242,40 @@ def up_ifaces(self, ifaces):
         for i in ifaces:
             card = pyw.getcard(i.get_name())
             pyw.up(card)
+    
+    def change_ifaces_mac(self, ifaces, mac_addrs):
```

Comment:
Also for this method it would be better to do it only for one interface at the time. This leads to the code being more readable since this method is called twice at max.